Class/Warrior/TalentTrees/Protection.cpp

```diff
@@ -107,3 +110,16 @@ void Protection::add_improved_shield_block(QMap<QString, Talent*>& talent_tier)
 
     add_talent_to_tier(talent_tier, talent);
 }
+
+void Protection::add_improved_sunder_armor(QMap<QString, Talent*>& talent_tier) {
+    QMap<unsigned, QString> rank_descriptions;
+    QString base_str = "Reduces the cost of your Sunder Armor ability by %1 rage.";
+    rank_descriptions.insert(0, base_str.arg(0));
```

Comment:
This should be arg(1), right now the talent preview says rank 1 reduces Sunder Armor cost by 0 rage.

Also, you don't need to explicitly rank all the rank descriptions like this. It is only necessary when there are non-linear increases, like the other example in this file.

Replace this function body with the following, which automatically formats the talent strings with the necessary linear increases described by the `QVector<QPair<unsigned, unsigned>> {{1, 1}}` argument (given on the form `{<start value>, <linear increase per talent point>}`).

```
auto talent = get_new_talent(warrior, "Improved Sunder Armor", "4LL", base_url + "ability/Ability_warrior_sunder.png", 3, "Reduces the cost of your Sunder Armor ability by %1 rage.", QVector<QPair<unsigned, unsigned>> {{1, 1}}, QVector<SpellRankGroup*> {spells->get_spell_rank_group_by_name("Sunder Armor")});

add_talent_to_tier(talent_tier, talent);
```